exercises/00-Environment-Setup/README.md

```diff
@@ -93,12 +67,14 @@ Navigate to `http://localhost:8080` in your web browser. Follow these steps:
 1.  **Enter the initial admin password**: You can get this from the jenkins container by running the command below.
 
 ```bash
-sudo docker exec jenkins-jdk-17 cat /var/jenkins_home/secrets/initialAdminPassword
+docker exec jenkins cat /var/jenkins_home/secrets/initialAdminPassword
 ```
 
 2.  **Install the Reccomended Plugins**: You'll install more later, but these are a good start for now.
 3.  **Create a new Admin user**: Enter your details, choose an admin password, and save.
 
 Congratulations! You have now set up a Jenkins server running on Docker Compose. This is a basic example to get you started with the exercises locally, and you will be learning more about this setup and making changes to it throughout the course.
 
+To shut these down at anytime, run `docker compose down` from this directory. 
+
 For now, go ahead and let the instructor know you're done.
```

Comment:
after doing the admin account setup, i had this page. there's no instructions on what to do from this step 

<img width="1033" alt="image" src="https://github.com/user-attachments/assets/5689e101-a3aa-4950-bee7-f0984d915712">

Reply:
Fixed in https://github.com/devbyaccident/Jenkins-level-1/pull/1/commits/c07266478e5092fb1fb3b1c8382ecc8f3dec7baa